src/Http/Http.Extensions/gen/RequestDelegateGeneratorSources.cs

```diff
@@ -42,6 +44,32 @@ public GeneratedProducesResponseTypeMetadata(Type? type, int statusCode, string[
 
         public IEnumerable<string> ContentTypes { get; }
     }
+
+    file sealed class GeneratedAcceptsMetadata : IAcceptsMetadata
```

Comment:
What do we think about making [our shared AcceptsMetadata](https://github.com/dotnet/aspnetcore/blob/b92218ab0c28465d5818871cd9e6fcf6956ca6da/src/Shared/RoutingMetadata/AcceptsMetadata.cs) public? We use it a lot ourselves at least.

Reply:
I am open to this idea.

Reply:
@halter73 I wrote up an API review for this: https://github.com/dotnet/aspnetcore/issues/47994

Reply:
Thank you! We didn't get to it in today's regularly scheduled API review, but hopefully we'll get to it soon. Obviously, this PR does not need to wait on the public API. It can continue generating the implementation for now.